includes/admin/upgrades/v3/class-data-migrator.php

```diff
@@ -475,7 +475,7 @@ public static function orders( $data = null ) {
 			'ip'             => $ip,
 			'gateway'        => $gateway,
 			'mode'           => $mode,
-			'currency'       => $payment_meta['currency'],
+			'currency'       => ! empty( $payment_meta['currency'] ) ? $payment_meta['currency'] : edd_get_curreny(),
```

Comment:
```suggestion
			'currency'       => ! empty( $payment_meta['currency'] ) ? $payment_meta['currency'] : edd_get_currency(),
```

Just a typo in the function name.

Reply:
I can't believe I missed that @ashleyfae, thanks!